src/utils/getNextStepper/getNextStepper.ts

```diff
@@ -0,0 +1,21 @@
+import type { TreeNode } from "@/types/TreeNode";
+
+const getNextStepper = (restArray?: TreeNode[]) => {
+  if (!restArray?.length) {
+    return 0;
+  }
+
+  let stepper = 0;
+
+  restArray.some(({ attributes: { type } }) => {
```

Comment:
Le some comme ça c'est pas ouf si tu laisse un let. C'est mieux avec le for je voyais pas ça désolé